src/Network/Receive/ServerType0.pm

```diff
@@ -2801,6 +2802,8 @@ sub private_message_sent {
 sub sync_received_characters {
 	my ($self, $args) = @_;
 
+	return unless (UNIVERSAL::isa($net, 'Network::DirectConnection'));
```

Comment:
Why?

Reply:
If we permit openkore use this sub we will get duplicated sync in character server, after this we get dc